note/witc_abi.md

```diff
@@ -0,0 +1,63 @@
+# Witc ABI
```

Comment:
I think this document should directly be the README of this folder

Reply:
Ok, moved to `note/README.md`